Sonic/Message/MsgDeactivateLayer.h

```diff
@@ -0,0 +1,17 @@
+﻿#pragma once
+
+#include <Hedgehog/Universe/Engine/hhMessage.h>
+
+namespace Sonic::Message
+{
+	class MsgDeactivatelayer : public Hedgehog::Universe::MessageTypeSet
+	{
+	public:
+		HH_FND_MSG_MAKE_TYPE(0x0167F0D4);
+		Hedgehog::base::CSharedString m_SetLayerName;
+		MsgDeactivatelayer(const Hedgehog::base::CSharedString& in_SetLayerName) : m_SetLayerName(in_SetLayerName) {}
```

Comment:
base -> Base
in_SetLayerName -> in_rSetLayerName

Reply:
Fixed, and you were right it has a capital L. Whoops.